resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/ErroredRuntimeChecker.scala

```diff
@@ -47,7 +47,9 @@ object ErroredRuntimeChecker {
                   )
                   .as(Some(runtime))
               else
-                logger.warn(s"${runtime} still exists in ${cluster.getStatus.getState.name()} status. Going to delete") >> deps.dataprocService
+                logger.warn(
```

Comment:
did you run scalafmt? I wouldn't format anything manually, and just leave formatting to scalafmt

Reply:
Yes, I had run `scalafmtAll`. Will run again after addressing the PR feedback.